src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int size = 6;
+        FigureDrawable[] figures = new FigureDrawable[size];
+
+        for (int i = 0; i < size / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = size / 2; i < size; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
+
+        for (FigureDrawable figure: figures) {
+            figure.getInfo();
```

Comment:
The `getInfo()` method is called, but its result is not used. If it's intended to print the information to the console, you should call a method that prints the information, like `System.out.println(figure.getInfo());`, assuming `getInfo()` returns a string. If `getInfo()` itself performs the printing, consider renaming it to reflect that action, such as `printInfo()` to improve readability.